### ISSUE TITLE
Ships get bugged when are attacked with ship-based weapons and VERY long reload

### ISSUE THREAD

**Issue by ABanC1001** [SuperGO2/supergo2-issues#74]
Description:
Also, the ship-based weapons last like 8 turns to reload, is a lot of time (if I remember it should be like 3-4) but after that it will reload every turn.
Also the fleet configuratión is not working, if I config max range - max attack, that doesn't happen.

How to replicate: 
I did it in instance 2, with the composition shown in screenshots

Screenshots:
![image](https://user-images.githubusercontent.com/108493836/176916466-8fd9d322-9c2d-46d9-8911-4166c3858858.png)

![image](https://user-images.githubusercontent.com/108493836/176916535-f83c3fbc-9a60-4d0b-9497-c5a0838aab66.png)

Edit: This was tested in instance 2

**Comment by ABanC1001** [comment]
I want to clarify that it was tested in instance 2

**Comment by ToxicSSJ** [comment]
@ABanC1001 I can't reproduce the "VERY long reload" part. I tested a similar design with Golem and it takes 4 rounds. Remember that there is no scientific research for weapons, so that could make you remember that this type of weapons have less CD than they really have.

This is the weapon that I tested:
![image](https://user-images.githubusercontent.com/36933597/177104072-90e22df6-05ff-45c5-bc78-fd62d9b7c3a3.png)

And in my test I sent three fleets to the instance 1, and one of them just fire only one time because the min range is 6 and the movement was just 2, and when the enemy fleet approached to 1 cell of distance it made my fleet was being unavailable to shoot for the rest of the combat.
